bin/packages/post-css-config.js

```diff
@@ -0,0 +1,64 @@
+module.exports = [
+	require( '../../packages/postcss-themes/src/index.js' )( {
```

Comment:
What if `src/index.js` contains code not supported by Node native? We're assumed to be able to use modern features there, and this may subtly break in the future.

Reply:
We can use `@wordpress/postcss-themes` since we started using local packages.